charts/seqr/templates/_helpers.tpl

```diff
@@ -56,7 +56,7 @@ Seqr environment shared between application and cron
 {{- define "seqr.environment" -}}
 envFrom:
   - configMapRef:
-      name: {{ .Release.Name }}-environment
```

Comment:
`seqr.fullname` and `.Release.Name` may be the same, but not guaranteed!  The `ConfigMap` name is denoted [here](https://github.com/broadinstitute/seqr-helm/blob/main/charts/seqr/templates/environment.yaml#L4)

Reply:
👍